docker/pypi/wmagent/install.sh

```diff
@@ -105,7 +116,8 @@ echo "-----------------------------------------------------------------------"
 stepMsg="Generating and preserving current build id"
 echo "-----------------------------------------------------------------------"
 echo "Start $stepMsg"
-echo $RANDOM |sha256sum |awk '{print $1}' > $WMA_ROOT_DIR/.dockerBuildId
+
+echo ${WMA_VER[release]}|sha256sum |awk '{print $1}' > $WMA_ROOT_DIR/.dockerBuildId
```

Comment:
To make it more readable, I would suggest whitespaces around the pipe operation.